main.py

```diff
@@ -0,0 +1,124 @@
+#info from https://scikit-learn.org/stable/modules/neighbors.html
+from sklearn import neighbors
+#quality is between 0-10 but as tested with the random guesses, 4-9 seems to make the result better
+#this is one ill definitely want to look at for hyperparemeter optimization
+knn = neighbors.KNeighborsRegressor() #6 neighbors for better results
+knn.fit(x_train, y_train)
+y_pred = knn.predict(x_test)
+acc = accuracy(y_pred, y_test)
+
+#print("\nMean Squared Error for Nearest Neighbor Regression:", mean_squared_error(y_test, y_pred))
+print("\nCoefficient of determination for Nearest Neighbor Regression:", r2_score(y_test, y_pred))
+print("Percent accuracy for Nearest Neighbor Regression: {0:.2f}%".format(acc))
+print("Absolute error score for Nearest Neighbor Regression:", d2_absolute_error_score(y_test, y_pred))
+
+############################################################################################################################################################################
+#support vector machine applied to regression
+############################################################################################################################################################################
+#code/info from https://scikit-learn.org/stable/modules/svm.html#regression
+from sklearn import svm
+supportvm = svm.SVC()
+supportvm.fit(x_train, y_train)
+y_pred = supportvm.predict(x_test)
+acc = accuracy(y_pred, y_test)
+
+#print("\nMean Squared Error for Support Vector Machine:", mean_squared_error(y_test, y_pred))
+print("\nCoefficient of determination for Support Vector Machine:", r2_score(y_test, y_pred))
+print("Percent accuracy for Support Vector Machine: {0:.2f}%".format(acc))
+print("Absolute error score for Support Vector Machine:", d2_absolute_error_score(y_test, y_pred))
+
+############################################################################################################################################################################
+#gradient boosting regression
+############################################################################################################################################################################
+#https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingRegressor.html#sklearn.ensemble.GradientBoostingRegressor
+from sklearn.ensemble import GradientBoostingRegressor
+gbr = GradientBoostingRegressor()
+gbr.fit(x_train, y_train)
+y_pred = gbr.predict(x_test)
+acc = accuracy(y_pred, y_test)
+
+#print("\nMean Squared Error for Gradient Boosting Regression:", mean_squared_error(y_test, y_pred))
+print("\nCoefficient of determination for Gradient Boosting Regression:", r2_score(y_test, y_pred))
+print("Percent accuracy for Gradient Boosting Regression: {0:.2f}%".format(acc))
+print("Absolute error score for Gradient Boosting Regression:", d2_absolute_error_score(y_test, y_pred))
```

Comment:
You can improve your code by putting this in a loop rather than repeating the same pattern.